app/src/main/java/com/example/datastoreapp/viewmodel/SharedPreferencesViewModel.kt

```diff
@@ -1,13 +1,22 @@
 package com.example.datastoreapp.viewmodel
 
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import com.example.datastoreapp.storetype.SharedPreferencesStore
 
-class SharedPreferencesViewModel(private val sharedPreferencesStorage: SharedPreferencesStore) : ViewModel() {
+class SharedPreferencesViewModel(private val sharedPreferencesStorage: SharedPreferencesStore) :
```

Comment:
You are going to have one ViewModel defined per each screen/type of store?

Reply:
I think she will need it to manage the different storages